inference-engine/tests/ie_test_utils/common_test_utils/common_utils.hpp

```diff
@@ -13,11 +13,15 @@ namespace CommonTestUtils {
 
 template<typename vecElementType>
 inline std::string vec2str(const std::vector<vecElementType> &vec) {
-    std::ostringstream result;
-    result << "(";
-    std::copy(vec.begin(), vec.end() - 1, std::ostream_iterator<vecElementType>(result, "."));
-    result << vec.back() << ")";
-    return result.str();
+    if (vec.size() > 0) {
+        std::ostringstream result;
+        result << "(";
+        std::copy(vec.begin(), vec.end() - 1, std::ostream_iterator<vecElementType>(result, "."));
+        result << vec.back() << ")";
+        return result.str();
+    } else {
```

Comment:
we can use only `return std::string("()");` without else. It will be more clear

Reply:
Agree< fixed